tests/charm++/dynamic_array_expansion/dynamical_insertion.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
```

Comment:
`#pragma once` is non-standard, it should ideally be replaced with a more portable solution,